internal/errcheck/errcheck.go

```diff
@@ -108,6 +110,12 @@ type Checker struct {
 	// If true, checking of _test.go files is disabled
 	WithoutTests bool
 
+	//Only check a single file
+	OnlyCheck string
+
+	//Extra output (to help distinguish output from other linters)
```

Comment:
Minor style nit. There should be a space after `//`. See https://dmitri.shuralyov.com/idiomatic-go#comments-for-humans-always-have-a-single-space-after-the-slashes.